.travis.yml

```diff
@@ -3,7 +3,6 @@ language: go
 go_import_path: go.uber.org/fx
 
 go:
-  - 1.7
```

Comment:
Completely unrelated to the proposal. I'm ok with the change, but this can be insta-merged in another PR with reasoning as to why we drop 1.7